apps/docs/stories/components/ModularCheckout/introduction.mdx

```diff
@@ -0,0 +1,173 @@
+
+### 3. Validation Orchestration
+
+When validation is triggered, the wrapper:
+
+- Validates all payment method forms (card, bank account)
+- Validates billing information forms
+- Validates specialized components (insurance, etc.)
+- Returns a combined validation result
+
+### 4. Tokenization Process
+
+For checkout submission, the wrapper:
+
+- Collects data from billing forms
+- Combines billing data with payment method data
+- Tokenizes payment methods securely
+- Handles different payment flows (cards, bank accounts, BNPL)
+
+### 5. Checkout Completion
+
+The final step involves:
+
+- Submitting the tokenized payment data
+- Handling success/error responses
+- Emitting completion events
+
+## Key Benefits
+
+### Flexibility
+
+Build exactly the checkout experience you need by combining only the components you require.
+
+### Security
+
+Payment-sensitive data is handled through secure tokenization, with card details processed in isolated iframes.
+
+### Maintainability
+
+Each component has a single responsibility, making the system easier to understand and maintain.
+
+### Extensibility
+
+New payment methods or features can be added as individual components without modifying existing code.
+
+## Common Usage Patterns
+
+### Basic Card Checkout
+
+```html
+<justifi-modular-checkout
+  auth-token="123xyz"
+  account-id="acc_1234567890"
+  checkout-id="cho_1234567890"
+>
+  <justifi-checkout-summary />
+  <justifi-card-form />
+  <justifi-billing-information-form />
+</justifi-modular-checkout>
+```
+
+### Saved Payment Methods with New Card Option
+
+```html
+<justifi-modular-checkout
+  auth-token="123xyz"
+  account-id="acc_1234567890"
+  checkout-id="cho_1234567890"
+>
+  <justifi-checkout-summary />
+  <justifi-saved-payment-methods />
+  <justifi-card-form />
+  <justifi-postal-code-form />
+</justifi-modular-checkout>
+```
+
+### Multi-Payment Method Checkout
+
+```html
+<justifi-modular-checkout
+  auth-token="123xyz"
+  account-id="acc_1234567890"
+  checkout-id="cho_1234567890"
+>
+  <justifi-checkout-summary />
+  <justifi-saved-payment-methods />
+  <justifi-card-form />
+  <justifi-bank-account-form />
+  <justifi-sezzle-payment-method />
+  <justifi-billing-information-form />
+  <justifi-season-interruption-insurance />
+</justifi-modular-checkout>
+```
+
+## Events and Methods
+
+### Events
+
+- **`error-event`**: Emitted when validation fails or errors occur during processing
+- **`checkout-complete-event`**: Emitted when checkout is successfully completed
+
+### Methods
+
+- **`validate()`**: Validates all child components and returns a boolean result
+- **`submitCheckout(billingInfo?)`**: Submits the checkout with optional billing information override
+
+## Getting Started
+
+To begin using the Modular Checkout system:
+
+1. Review the individual component documentation in the sections below
+2. Choose the components that match your requirements
```

Comment:
I think we need these 2 steps in here:
3. Create a checkout via create checkout API
4. Create a web component token

with added link to https://docs.justifi.tech/api-spec#tag/Checkouts/operation/CreateCheckout
and to the Authorization section